netlify/netlify-plugin-nextjs
# chore: add nx demo

### Summary

Adds an nx demo site and cypress config specific to it

<img width="350" alt="" src="https://user-images.githubusercontent.com/11183523/144041513-16da16d5-3824-4022-af04-f175cec020c8.png">

### Test plan

1. Visit the Deploy Preview: https://deploy-preview-854--netlify-plugin-nextjs-nx-monorepo-demo.netlify.app/
2. Check logs and see it is running the cypress config for nx: https://app.netlify.com/sites/netlify-plugin-nextjs-nx-monorepo-demo/deploys/61a60ef6e913a116996cad14

### Relevant links (GitHub issues, Notion docs, etc.) or a picture of cute animal

#839 

### Standard checks:




---

🧪 Once merged, make sure to update the version if needed and that it was published correctly.
